Makefile

```diff
@@ -0,0 +1,53 @@
+####### Universal ######
+
+pip:
+	pip install -e .
+
+pip-test: pip
```

Comment:
@jasonpaulos - per our conversation, I'm recommending going with the approach of having a special "test" requirements using `setup.py`'s `require_extras` dict.

Some brief research shows that there is [PEP 621](https://peps.python.org/pep-0621/#example) will give a different solution in the future, but AFAIK no version of `pip` yet supports this. Therefore, I'm going with the next best option, in my opinion, which is [discussed here](https://discuss.python.org/t/is-it-preferable-to-add-test-dependensies-as-an-extra-require-instead-of-tox-ini/4984?filter=summary).

Reply:
I can accept this, though I'd suggestion `development` or similar instead of just `test`. The dev process involves more than just tests, there are also linters, formatters, etc. that help us write code.